event_webhook_types.go

```diff
@@ -757,6 +757,10 @@ type MergeEvent struct {
 			Previous int `json:"previous"`
 			Current  int `json:"current"`
 		} `json:"updated_by_id"`
+		MergeStatus struct {
+			Previous string `json:"previous"`
+			Current  string `json:"current"`
+		}`json:"merge_status"`
```

Comment:
Can you put it in the alphabetical ordered location? Thanks!

Reply:
Oh. I didn't notice that, will create a new pr to update